schema/transportation/segment.yaml

```diff
@@ -34,13 +34,12 @@ properties:
         "$comment": >-
           Should not be confused with a transport mode. A segment
           kind has an (implied) set of default transport modes.
-      connectors:
-        "$comment": 'FIXME: Must rename to connectorIds to meet schema standard'
+      connector_ids:
         description: >-
-          List of connectors this segment is physically connected to.
-          Each connector is a possible routing decision point meaning it
-          defines place along segment in which there is possibility to
-          transition to other segments which share same connector.
+          List of connector IDs this segment is physically connected to.
+          Each connector is a possible routing decision point, meaning it
```

Comment:
If I put on my Uber-nitpicker x 1000 hat, I would put the phrase *identifying the connectors* after *connector IDs*, as in:

> List of connector IDs *identifying connectors* this segment is physically connected to ......

I argue that's slightly more conceptually correct, since abstractly, the segment is connected to the connectors, not their IDs.